### PR TITLE
Add CVE-2012-6153

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
@vasulakkaraju @GeetikaBatra this one should be easy :wink:

**Comment by vasulakkaraju** [comment]
Updated CVE info. Pls review.

File: database/java/2012/6153.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - https://bugzilla.redhat.com/show_bug.cgi?id=1129916
+    - https://h20566.www2.hpe.com/portal/site/hpsc/public/kb/docDisplay?docId=emr_na-c05103564
+
+affected:
+    - groupId: org.apache.httpcomponents
+      artifactId: httpclient
+      version:
+        - "<= 4.2.2"
+      fixedin:
+        - "== 4.2.3"
```
Comment (msrb) [inline database/java/2012/6153.yaml]: Whitespaces :) Also please use `">=4.2.3"` as newer versions automatically inherit the fix for the vulnerability.

**Comment by vasulakkaraju** [comment]
as these were updated yesterday, whitespace issue still existed,anyhow updated all the info.Thanks

**Comment by msrb** [comment]
@vasulakkaraju merging, thanks :wink:

**Pull request merged**